docs/contribute.rst

```diff
@@ -15,16 +15,16 @@ Setting up the development environment
 ======================================
 
 While there are many ways to set up one's development environment, the following
-instructions will utilize Pip_ and Poetry_. These tools facilitate managing
+instructions will utilize Pip_ and pdm_. These tools facilitate managing
```

Comment:
Let's use the canonical capitalized name when referring to PDM in all places (except when it's clearly a CLI invocation like `pdm install`).